### PR TITLE
feat: add action for test

### PR DESCRIPTION
**Pull request by AndresDFX**

### CODE REVIEW

File: .github/workflows/deploy_aws.yml [hunk 1]
```diff
@@ -12,13 +12,30 @@ env:
...
 #permissions:
 #  contents: read
 
+
 jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        directory: ${{fromJson(env.DIRECTORY)}}
```
Comment (DanielFCubides) [inline .github/workflows/deploy_aws.yml:27]: Please explain what does this strategy

File: .github/workflows/deploy_aws.yml [hunk 2]
```diff
@@ -12,13 +12,30 @@ env:
...
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Build Docker image from ${{ matrix.directory }}
+      run: |
+        docker build -t test-image-${{ matrix.directory }} ./${{ matrix.directory }}
```
Comment (DanielFCubides) [inline .github/workflows/deploy_aws.yml:35]: You are not running the Target test in this step

Comment (AndresDFX) [reply .github/workflows/deploy_aws.yml:35]: just by building the image I can know if the pytest failed

File: .github/workflows/deploy_aws.yml (hunk 1 above)
Comment (AndresDFX) [reply .github/workflows/deploy_aws.yml:27]: Using an array listing those directories can automate and simplify this process.

**Comment by DanielFCubides** [comment]
@AndresDFX please update the branch

**Pull request merged**